velox/docs/functions/spark/array.rst

```diff
@@ -112,6 +112,26 @@ Array Functions
         SELECT array_sort(array(NULL, 1, NULL)); -- [1, NULL, NULL]
         SELECT array_sort(array(NULL, 2, 1)); -- [1, 2, NULL]
 
+.. spark:function:: array_sort(array(T), function(T,U)) -> array(T)
+    :noindex:
+
+    Returns the array sorted by values computed using specified lambda in ascending
+    order. U must be an orderable type. Null elements will be placed at the end of
+    the returned array. ::
+
+        SELECT array_sort(array('cat', 'leopard', 'mouse'), x -> length(x)); -- ['cat', 'mouse', 'leopard']
+
+.. spark:function:: array_sort(array(T), function(T,T,U)) -> array(T)
```

Comment:
I only find below signature is registered. Am I miss something?
> array_sort_desc(array(T), function(T,U))

Reply:
It's here:
https://github.com/facebookincubator/velox/blob/0093ee9348c705d460a204aef9e17e595aedfcce/velox/functions/sparksql/Register.cpp#L331